collectors/proc.plugin/proc_net_dev.c

```diff
@@ -659,6 +693,39 @@ int do_proc_net_dev(int update_every, usec_t dt) {
             d->tcarrier    = str2kernel_uint_t(procfile_lineword(ff, l, 15));
         }
 
+        if (d->do_duplex != CONFIG_BOOLEAN_NO && d->filename_duplex) {
+            char buffer[32 + 1];
```

Comment:
I will give a suggestion here,  that I received in other PRs, please, instead to use `32 + 1`, add a `define` that will give a meaningful name.